modules/pulsar/src/main/java/org/testcontainers/containers/PulsarContainer.java

```diff
@@ -0,0 +1,44 @@
+package org.testcontainers.containers;
+
+import org.testcontainers.utility.Base58;
+import org.testcontainers.utility.TestcontainersConfiguration;
+
+import java.util.stream.Stream;
+
+/**
+ * This container wraps Apache pulsar running in stanalone mode
+ */
+public class PulsarContainer extends GenericContainer<PulsarContainer> {
+
+    public static final int PULSAR_PORT = 6850;
+
+    public PulsarContainer() {
+        this("2.0.0-rc1-incubating");
+    }
+
+    public PulsarContainer(String pulsarVersion) {
+        super(TestcontainersConfiguration.getInstance().getPulsarImage() + ":" + pulsarVersion);
+
+        withNetwork(Network.newNetwork());
+        String networkAlias = "pulsar-" + Base58.randomString(6);
+        withNetworkAliases(networkAlias);
+        withExposedPorts(PULSAR_PORT);
```

Comment:
network is not needed here (was only needed for Kafka), only `withExposedPorts`

Reply:
done